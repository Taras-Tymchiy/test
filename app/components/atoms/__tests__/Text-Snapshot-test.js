import React from 'react';
import renderer from 'react-test-renderer';
import { 
  TextRegular,
  TextMuted,
  TextPrimary,
  TextSecondary,
  TextHeader
 } from '../Text';

 it('renders a TextRegular using Snapshots', () => {
  expect(renderer.create(
    <TextRegular>TextRegular</TextRegular>
  )).toMatchSnapshot();
});

it('renders a TextMuted using Snapshots', () => {
  expect(renderer.create(
    <TextMuted>TextMuted</TextMuted>
  )).toMatchSnapshot();
});

it('renders a TextPrimary using Snapshots', () => {
  expect(renderer.create(
    <TextPrimary>TextPrimary</TextPrimary>
  )).toMatchSnapshot();
});

it('renders a TextSecondary using Snapshots', () => {
  expect(renderer.create(
    <TextSecondary>TextSecondary</TextSecondary>
  )).toMatchSnapshot();
});

it('renders a TextHeader using Snapshots', () => {
  expect(renderer.create(
    <TextHeader>TextHeader</TextHeader>
  )).toMatchSnapshot();
});

