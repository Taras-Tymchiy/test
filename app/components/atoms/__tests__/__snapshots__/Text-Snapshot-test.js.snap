// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders a TextHeader using Snapshots 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#222",
        "fontSize": 20,
        "fontWeight": "bold",
      },
      undefined,
    ]
  }
>
  TextHeader
</Text>
`;

exports[`renders a TextMuted using Snapshots 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#A0A0A0",
        "fontSize": 16,
        "fontWeight": "normal",
      },
      undefined,
    ]
  }
>
  TextMuted
</Text>
`;

exports[`renders a TextPrimary using Snapshots 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#222",
        "fontSize": 16,
        "fontWeight": "bold",
      },
      undefined,
    ]
  }
>
  TextPrimary
</Text>
`;

exports[`renders a TextRegular using Snapshots 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#222",
        "fontSize": 16,
        "fontWeight": "normal",
      },
      undefined,
    ]
  }
>
  TextRegular
</Text>
`;

exports[`renders a TextSecondary using Snapshots 1`] = `
<Text
  accessible={true}
  allowFontScaling={true}
  ellipsizeMode="tail"
  style={
    Array [
      Object {
        "color": "#A0A0A0",
        "fontSize": 14,
        "fontWeight": "normal",
      },
      undefined,
    ]
  }
>
  TextSecondary
</Text>
`;
